static/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,10 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-unsubscribed">
+                    <span>No streams to show</span>
+                    <a href="#streams/all">View all streams</a>
+                </div>
```

Comment:
I'd have expected this to be next to whatever we display when one of the existing panels are empty; is the story that we currently don't display anything when those tabs are empty?

Reply:
I suppose we never have an empty list for the `Subscribed` and `All streams` tab?

Reply:
I'm pretty sure it's possible, but a very unlikely situation, so it'd be reasonable to add a similar notice for that case in its own commit.